osf/migrations/0162_conference_submissions.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-06-08 15:11
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0161_add_spam_fields_to_user'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='conference',
+            name='submissions',
+            field=models.ManyToManyField(related_name='conferences', to='osf.AbstractNode'),
+        ),
+        migrations.RemoveField(
+            model_name='conference',
+            name='num_submissions',
```

Comment:
No need to cache `num_submissions` field any longer. We have a `submission` field!